packages/terra-spacer/tests/nightwatch/SpacerLarge+1.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './SpacerTest.scss';
+import Spacer from '../../lib/Spacer';
+
+export default () => (
+  <div>
+    <div className="spacing-demo spacer-demo-margin">
+      <Spacer id="spacermarginlarge1" className="spacer" marginTop="large+1" marginBottom="large+1" marginLeft="large+1" marginRight="large+1">
```

Comment:
nit: I've noticed you've done this in a few examples now, can you use camel case when providing classNames or ids? Then `spacermarginlarge1` would appear as `spacerMarginLarge1` which is a bit more readable with longer names.

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/0b85f9f949ade6f12fb384a5fff8facfaa91784b#diff-c3c6c20b48490f97705d4b284e95f159R5